src/software/world/field.cpp

```diff
@@ -141,6 +141,11 @@ double Field::centerCircleRadius() const
     return center_circle_radius_;
 }
 
+double Field::goalToPenaltyMark() const
+{
+    return (double)2 / 3 * xLength();
+}
```

Comment:
nit: you could probably make this a constant the field class owns?

This is currently public but it seems like a helper?

Reply:
updated